README.md

```diff
@@ -19,13 +19,13 @@ Any feedback on how it works on other Acer laptops would be appreciated.
 ## Building
 
 Make sure that you have the kernel headers for your kernel installed
-and type `make` in the cloned project directory. In more detail,
+and type `make KERNELVERSION=$(uname -r)` in the cloned project directory. In more detail,
```

Comment:
The build instructions should not need to change once you adapt the Makefile as described above.